### ISSUE TITLE
Feedback on Cypress tests

### ISSUE THREAD

**Issue by dmundra** [CivicActions/accessibility#478]
Feedback from @jonbot on the [Cypress tests](https://github.com/CivicActions/accessibility/tree/main/cypress)

Overall, this looks good, and I like how you've made a Command out of `checkA11yWithMultipleViewPorts`.

Some suggestions/things to think about:

1. Right now you have a spec for your home page, but you may eventually want to grab all of your pages (such as from a sitemap) and iterate through them. What you have makes perfect sense for now, but you may want to organize a bit differently at some point.
2. Nitpick/suggestion: do you need `Cypress._.isArray()`? It looks like built-in `Array.isArray()` is compatible with modern browsers. NBD, but it may run faster than lodash.
3. You might want to add some arguments to `cy.checkA11y()`, particularly a violation callback which is essential in a CI env. There's a perfect example on [cypress-axe's GitHub page](https://github.com/component-driven/cypress-axe#using-the-violationcallback-argument).
4. In your search spec, I'm curious if you need to wait for results before checking accessibility.
5. Finally, it looks like you're missing `axe-core` in your package.json.

**Comment by dmundra** [comment]
For 4, I was watching the Cypress live run and it does seem to wait but I think I can also guarantee it, so good point. For 5, axe-core already existed in the lock file for pa11y and I don’t think it needs to be specified as it should be there and we don’t directly call it (yet).

**Comment by dmundra** [comment]
For search, it is all client-side so there are no additional traffic requests made to wait so the search test should be fine. https://accessibility.civicactions.com/search
